tamr_toolbox/workflow/jobs.py

```diff
@@ -188,3 +188,26 @@ def get_upstream_projects(project: Project) -> List[Project]:
     upstream_projects = _find_upstream_projects(project)
 
     return upstream_projects
+
+
+def get_project_output_datasets(project: Project) -> List[Dataset]:
+    """Retrieves datasets produced by a given Tamr project
+    Args:
+        project: the Tamr project for which associated output datasets are retrieved
+
+    Returns:
+        The list of Tamr datasets output from the project"""
```

Comment:
Minor docstring formatting:
```suggestion
    """Retrieves datasets produced by a given Tamr project
    
    Args:
        project: the Tamr project for which associated output datasets are retrieved

    Returns:
        The list of Tamr datasets output from the project
    """
```